src/Client/Resources/keen-io-3_0.php

```diff
@@ -40,22 +40,72 @@
                     'type'        => 'string',
                     'required'    => true,
                 ),
-                'project_data' => array(
+                'name' => array(
+                    'location' => 'json',
+                    'type'     => 'string',
+                    'required' => true,
+                ),
+                'users' => array(
+                    'location' => 'json',
+                    'type'     => 'array',
+                    'required' => true,
+                ),
+                'preferences' => array(
+                    'location' => 'json',
+                    'type'     => 'object',
+                ),
+            ),
+        ),
+
+        'updateProject' => array(
+            'uri'         => 'organizations/{organizationId}/projects/{projectId}',
+            'description' => 'Updates a project for the specified organization.',
+            'httpMethod'  => 'POST',
+            'parameters'  => array(
+                'organizationId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'projectId'  => array(
+                    'location'    => 'uri',
+                    'type'        => 'string'
+                ),
+                'organizationKey' => array(
+                    'location'    => 'header',
+                    'description' => 'The Organization Key.',
+                    'sentAs'      => 'Authorization',
+                    'pattern'     => '/^([[:alnum:]])+$/',
+                    'type'        => 'string',
+                    'required'    => true,
+                ),
+                'name' => array(
+                    'location' => 'json',
+                    'type'     => 'string',
+                ),
+                'users' => array(
                     'location' => 'json',
                     'type'     => 'array',
                 ),
+                'preferences' => array(
+                    'location' => 'json',
+                    'type'     => 'object',
+                ),
             ),
         ),
 
         'getProjects' => array(
-            'uri'         => 'projects',
+            'uri'         => 'organizations/{organizationId}/projects',
             'description' => 'Returns the projects accessible to the API user, as well as '
```

Comment:
Maybe this should be updated to `'Returns information about all projects for the given organization.'` to match https://keen.io/docs/api/#get-projects